targets/TARGET_STM/TARGET_STM32F0/lp_ticker_defines.h

```diff
@@ -0,0 +1,28 @@
+/* mbed Microcontroller Library
+ * SPDX-License-Identifier: BSD-3-Clause
+ ******************************************************************************
+ *
+ * Copyright (c) 2015-2020 STMicroelectronics.
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ ******************************************************************************
+ */
+
+#ifndef __LP_TICKER_DEFINES_H
+#define __LP_TICKER_DEFINES_H
+
+#define LP_TICKER_PERIOD_NUM    1000000
+
+#if MBED_CONF_TARGET_LSE_AVAILABLE
+#define LP_TICKER_PERIOD_DEN    (LSE_VALUE / 4) // RTC_CLOCK / RTC_WAKEUPCLOCK_RTCCLK_DIV4
```

Comment:
What are the comments trying to say here?

If that's a better way of writing the expression, why not use it?

If you can't due to non-visibility of those macros in general context, so you're writing something you think should be equivalent, how about asserting equivalence in a C file where you can?

    #if LP_TICKER_PERIOD_DEN != RTC_CLOCK / RTC_WAKEUPCLOCK_RTCCLK_DIV4
    #error "Mismatch with lp_ticker_defines.h"
    #endif
    
Although I guess the lp_ticker_test would catch it with its runtime cross-check.